### ISSUE TITLE
The range of target table has an unexpected number of replicas during CSV files import

### ISSUE THREAD

**Issue by heidawei** [cockroachdb/cockroach#51451]
**Describe the problem**

Please describe the issue you observed, and any steps we can take to reproduce it:
I set database num replicas to 1.
`ALTER DATABASE db0 CONFIGURE ZONE USING num_replicas = 1, range_min_bytes = 134217728, range_max_bytes = 536870912, gc.ttlseconds=7200;`

then I import table1, the CSV files total size is 1.5 TB, import is fast.
I import table2, the CSV files(number of csv files more than the number of cluster nodes) total size is 1.6TB, I find the import is slow, and the num replicas per node is weird, the cluster USED CAPACITY is several times more than before import the table 2, when table 2 import finish, the cluster start clear replicas. I see the num replicas per node is decline.

**To Reproduce**

What did you do? Describe in your own words.

If possible, provide steps to reproduce the behavior:

1. Set up CockroachDB cluster ...
2. Send SQL ... / CLI command ...
3. Look at UI / log file / client app ...

**Expected behavior**

**Additional data / screenshots**
If the problem is SQL-related, include a copy of the SQL query and the schema
of the supporting tables.

If a node in your cluster encountered a fatal error, supply the contents of the
log directories (at minimum of the affected node(s), but preferably all nodes).

Note that log files can contain confidential information. Please continue
creating this issue, but contact support@cockroachlabs.com to submit the log
files in private.

**Environment:**
 - CockroachDB version [e.g. 2.0.x]
   V 19.2
 - Server OS: [e.g. Linux/Distrib]
 - Client app [e.g. `cockroach sql`, JDBC, ...]

**Additional context**
What was the impact?

**Comment by dt** [comment]
Hey KV folks, is this expected during scatter? part of atomic replica stuff?

**Comment by heidawei** [comment]
If I set `ALTER RANGE default CONFIGURE ZONE USING num_replicas = 1`, the table2 import is fast and the number of replicas is normal, It must be noted that both table1 and table 2 belong to the database db0 that I tested.

In the previous strange test phenomenon, check that the replica configuration of db0 and table1 is 1
It is easy to replicate this phenomenon by continuously importing two large tables into the cluster with the database replica configured as 1. In my test, the size of the two tables is 1.5TB. They are not a file, but a file collection, mainly to improve the import performance.
